common-arcana.lic

```diff
@@ -60,8 +60,13 @@ module DRCA
   def buff(spell, settings)
     prepare?(spell['abbrev'], spell['mana'], spell['symbiosis'])
     find_charge_invoke_stow(settings.cambrinth, settings.stored_cambrinth, settings.cambrinth_cap, settings.dedicated_camb_use, spell['cambrinth'])
-    waitcastrt?
-    cast?(spell['cast'], spell['symbiosis'], spell['before'], spell['after'])
+
+    if spell['ritual']
+      ritual(spell)
```

Comment:
It looks like prepare and cambrinth are done before this check and also done in `ritual`. Does that mean it will attempt to prepare, cambrinth, prepare, focus?